omniledger/service/api.go

```diff
@@ -6,17 +6,24 @@ package service
  */
 
 import (
+	"errors"
+
 	"github.com/dedis/student_18_omniledger/omniledger/darc"
 
 	"gopkg.in/dedis/cothority.v2"
 	"gopkg.in/dedis/cothority.v2/skipchain"
-	"gopkg.in/dedis/kyber.v2"
 	"gopkg.in/dedis/onet.v2"
 )
 
 // ServiceName is used for registration on the onet.
 const ServiceName = "OmniLedger"
 
+// ActionAddDarc is the action name for adding a darc to OmniLedger.
+const ActionAddDarc = darc.Action("add_darc")
```

Comment:
if we take `kind`_`command`, this would be:
- `create_darc`
- `create_genesis`